dictionaries/telemetry/damageCauserName.json

```diff
@@ -74,6 +74,7 @@
   "Uaz_B_01_C": "UAZ (soft top)",
   "Uaz_B_01_esports_C": "UAZ (soft top)",
   "Uaz_C_01_C": "UAZ (hard top)",
+  "Uaz_Armored_C": "UAZ (armored)",
```

Comment:
Thank you! We have these in alphabetical order though so this should be on line 74.